exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
```

Comment:
Your function names in the code could look better. There are some functions that, as I understood, didn't match the documentation.

Please address it.